usage/usage.py

```diff
@@ -266,7 +266,7 @@ def get_args(args):
     return title, name, color, key_len, mapping, resolution
 
 
-def make_plot(title, times, nodes, name, job_list, color, key_len, mapping):
+def make_plot(title, times, nodes, name, job_list, color):
     """Makes a time vs. NNode plot named 'name'.png.
```

Comment:
I feel rather awkward trying to argue with you, an English native speaker, about your language, but shouldn't the summary be the other way round: "Plots number of nodes vs. time."? I may be totally wrong, but usually it is the dependent variable against the independent one.

Reply:
*facepalm* Yes, you're completely right!  Sometimes I accidentally forget to write it "y vs. x" and don't catch my mistakes.